qemu/tests/nvdimm_mode.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+from virttest import utils_package
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run nvdimm cases:
+    1) Boot guest with two nvdimm devices
+    2) Change the two nvdimm devices to dax mode inside guest
+    3) Check if both devices are dax mode
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+    utils_package.package_install("ndctl", session)
+    create_dax_cmd = params["create_dax_cmd"]
+    nvdimm_number = len(params["mem_devs"].split())
+    try:
+        for i in range(nvdimm_number):
+            session.cmd(create_dax_cmd % i)
+        output = session.cmd_output(params["ndctl_check_cmd"])
+        dax_number = len(re.findall('"mode":"devdax"', output))
```

Comment:
I prefer to check the key-value,more robust. 
```
output = eval(output)
for item in output:
    if item['mode'] != 'devdax':
        test.fail(****)     
```

It is up to you.

Reply:
> I am not familiar with the `align`. When set 128M in the command line, the output of `ndctl list` show it is `4096` . Is it normal ?

Yes, they have different meaning. The align inside guest should be the same value in 
`create_dax_cmd = "ndctl create-namespace -m dax -e namespace%s.0 -f -v -a 8192"`